packages/@o3r/analytics/ng-package.json

```diff
@@ -6,5 +6,6 @@
 
   "lib": {
     "entryFile": "src/public_api.ts"
-  }
+  },
+  "allowedNonPeerDependencies": ["tslib"]
```

Comment:
this is ok but it is not mandatory to specify it when it is tslib only :)

Reply:
I have seen that. I will remove it:)